### PR TITLE
Clarify loading of kubeconfig for in-cluster headlamp installations

### PR DESCRIPTION
**Pull request by gberche-orange**

### CODE REVIEW

File: docs/installation/in-cluster/index.md [hunk 1]
```diff
@@ -43,6 +43,18 @@ you can run:
 kubectl apply -f https://raw.githubusercontent.com/kinvolk/headlamp/main/kubernetes-headlamp.yaml
 ```
 
+## Use a non-default kube config file
+
+By default headlamp uses the default service account from the namespace it is deployed to, and generates a kubeconfig from it named `main`.
+
+If you wish to use another specific non-default kubeconfig file, then you can do it by mouting it to the default location at `/home/headlamp/.config/Headlamp/kubeconfigs/config`, or
```
Comment (illume) [inline docs/installation/in-cluster/index.md:50]: There's a typo with "mouting" that should be "mounting".

**Comment by illume** [comment]
Thanks! These are very good additions.

File: docs/installation/in-cluster/index.md [hunk 2]
```diff
@@ -43,6 +43,18 @@ you can run:
 kubectl apply -f https://raw.githubusercontent.com/kinvolk/headlamp/main/kubernetes-headlamp.yaml
 ```
 
+## Use a non-default kube config file
+
+By default headlamp uses the default service account from the namespace it is deployed to, and generates a kubeconfig from it named `main`.
```
Comment (skoeva) [inline docs/installation/in-cluster/index.md:48]:
```suggestion
By default Headlamp uses the default service account from the namespace it is deployed to, and generates a kubeconfig from it named `main`.
```

File: docs/installation/in-cluster/index.md (hunk 1 above)
Comment (skoeva) [inline docs/installation/in-cluster/index.md:50]:
```suggestion
If you wish to use another specific non-default kubeconfig file, then you can do it by mounting it to the default location at `/home/headlamp/.config/Headlamp/kubeconfigs/config`, or 
```

File: docs/installation/in-cluster/index.md [hunk 3]
```diff
@@ -43,6 +43,18 @@ you can run:
...
+
+If you wish to use another specific non-default kubeconfig file, then you can do it by mouting it to the default location at `/home/headlamp/.config/Headlamp/kubeconfigs/config`, or 
+providing a custom path Headlamp with the ` -kubeconfig` argument or the KUBECONFIG env (through helm values.env)
```
Comment (skoeva) [inline docs/installation/in-cluster/index.md:51]:
```suggestion
providing a custom path with the ` -kubeconfig` argument or the KUBECONFIG env (through helm values.env)
```

File: docs/installation/in-cluster/index.md [hunk 4]
```diff
@@ -43,6 +43,18 @@ you can run:
...
+
+### Use several kube config files
```
Comment (skoeva) [inline docs/installation/in-cluster/index.md:53]:
```suggestion
### Use several kubeconfig files
```

**Review by skoeva — COMMENTED** [review]
Left some minor suggested changes that should hopefully make this easier to revise. Thanks for this!

**Review by illume — APPROVED** [review]
👍 thanks

**Pull request merged**